skipper.go

```diff
@@ -1770,11 +1770,17 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 	}
 
 	if o.CloneRoute != nil {
-		ro.PreProcessors = append(ro.PreProcessors, o.CloneRoute)
+		for idx := range o.CloneRoute {
+			cr := o.CloneRoute[idx]
+			ro.PreProcessors = append(ro.PreProcessors, cr)
+		}
```

Comment:
Wouldn't this work?
```go
ro.PreProcessors = append(ro.PreProcessors, o.CloneRoute...)
```
if not it still could be simpler
```go
for _, cr := range o.CloneRoute {
	ro.PreProcessors = append(ro.PreProcessors, cr)
}
```

Reply:
The problem is that `routing.PreProcessor` is an interface type and `*eskip.Clone` is implementing the interface.
CloneRoute would have to be the interface type in case this ... thing would work as far as I understand.
If the second way works it's fine to use this one.